bindings/python/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2022 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+pybind11_add_module(pybind11_icub_models MODULE
+  ${CMAKE_CURRENT_SOURCE_DIR}/icub_models.cpp
+  )
+
+target_include_directories(pybind11_icub_models PUBLIC "$<BUILD_INTERFACE:${pybind_include_dirs}>")
+
+target_link_libraries(pybind11_icub_models PRIVATE
+  icub-models::icub-models)
+
+# # The generated Python dynamic module must have the same name as the pybind11
+# # module, i.e. `bindings`.
+set_target_properties(pybind11_icub_models PROPERTIES
+    LIBRARY_OUTPUT_DIRECTORY "${ICUB_MODELS_PYTHON_PACKAGE}"
+    OUTPUT_NAME "bindings")
+
+
+# Output package is:
```

Comment:
```suggestion
# Output module is:
```
(The Python module is `icub_models`, Python package is `icub-models`, according to Python guidelines)

Reply:
This is a folder containing a `__init__.py` file right? The real module is `bindings...`. Now, a package is technically any folder with a init file inside, and in terms of naming it is not the same as a pip / PyPI "package" (this is what defined in the metadata. Note that the metadata could have a name (i.e. package name `bar`) containing a python package `foo` so that you install `pip install bar` but you import with `import foo`. Not really user friendly and highly discouraged, but there are widely used packages that do not have exactly matching names). So, if I understood, calling this a package should be correct. Yeah, it's confusing :)

Reply:
I got it, see https://docs.python.org/3/tutorial/modules.html#packages .